### PR TITLE
DIP-60: Ultimate Solidity Championship (USC) - Solidity Programming Competition

### PR DESCRIPTION
**Pull request by fiveoceans-dev**
## Description
I am proposing a DIP-60 a coding event aiming to bring together solidity programmers.

## Checklist
 * [x] Correct markdown (e.g. headers with ##, header surrounded by ---)
 * [x] syntax check
 * [x] filename is DIP-X.md (with X being the correct DIP number)
 * [x] header contains (in this order)
   * [x] DIP
   * [x] Title
   * [x] Status
   * [x] Themes
   * [x] Tags
   * [x] Authors
   * [x] Resources Required (optional)
   * [x] Discussion (with link to devcon forum)
   * [x] Created (in the format year-month-day e.g 2020-4-20)
 * [x] contains these sections:
   * [x] Abstract
   * [x] Rationale
   * [x] Implementation
   * [x] Operational Requirements & Ownership
 * [x] PR title prefixed with DIP-X

### CODE REVIEW

File: DIPs/DIP-60.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+DIP: 60
+Title: Solidity UFC - Solidity Programming Competition
+Status: Draft
+Themes: Social
+Instances: [Devcon6]
```
Comment (ligi) [inline DIPs/DIP-60.md:6]:
```suggestion
Instances: ["Devcon7"]
```
this change - or you need a time machine and go back to bogota ;-)

File: DIPs/DIP-60.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+Tags: Event Operations, Event Production, Communications, Sponsorships
+Authors: Five (@fiveoceans_dev)
```
Comment (ligi) [inline DIPs/DIP-60.md:8]:
```suggestion
Authors: Five (@fiveoceans-dev)
```
github name assumed here

**Review by ligi — CHANGES_REQUESTED** [review]
Super nice proposal - just 2 nits in the comments.
Also wondering if it is legal to name it around UFC or if this is protected.
Maybe a more generic "Race" or something like this is safer. Need to consult legal with this.

**Review by ligi — CHANGES_REQUESTED** [review]
In the screenshots it is still UFC not USC

**Comment by fiveoceans-dev** [comment]
Thanks, I have removed all instances of the word 'UFC'.

**Review by fiveoceans-dev — COMMENTED** [review]
i pushed updates to removed all instances of the word ‘UFC’

**Review by ligi — APPROVED** [review]

**Pull request merged**